release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -2584,6 +2584,10 @@ To work around this issue, before you update to {product-title} {product-version
 
 * There is a disk discovery delay when attaching storage to workloads. (link:https://issues.redhat.com/browse/OCPBUGS-11149[*OCPBUGS-11149*])
 
+* When upgrading from {product-title} 4.12 to 4.13, the Mellanox NIC renames SR-IOV network node policies such as `ens7f0` to `ens7f0np0`. Renaming the network node policies prevents the `SriovNetworkNodePolicy` object from locating the original interface only when the `pfNames` field is used in the policy such as `ens7f0`. Consequently, virtual functions (VFs) cannot be created because the interface cannot be found.
```

Comment:
the Mellanox NIC will be renamed such as `ens7f0` to `ens7f0np0` after upgrade to rhel9 kernel. since SriovNetworkNodePolicy was using the original nic name(ens7f0) before upgrade. So after upgrade the VF may cannot be created because the interface cannot be found

The workarounds is edit the `SriovNetworkNodePolicy` CR and add another interface name. eg

Given the original `SriovNetworkNodePolicy` was created before upgrade. like below
```
apiVersion: sriovnetwork.openshift.io/v1
kind: SriovNetworkNodePolicy
metadata:
  name: cx6dx
  namespace: openshift-sriov-network-operator
spec:
  deviceType: netdevice
  nicSelector:
    deviceID: "101d"
    pfNames:
      - ens7f0
    vendor: '15b3'
  nodeSelector:
    feature.node.kubernetes.io/sriov-capable: 'true'
  numVfs: 4
  mtu: 9000
  priority: 99
  resourceName: cx6dx
```

After upgrade if find the interface name `ens7f0` renamed to `ens7f0np0`. here we can edit the SriovNetworkNodePolicy CR by `oc edit SriovNetworkNodePolicy cx6dx` and then add another interface name `ens7f0np0`. 

```
  deviceType: netdevice
  nicSelector:
    deviceID: "101d"
    pfNames:
      - ens7f0
      - ens7f0np0
    vendor: '15b3'
  nodeSelector:
    feature.node.kubernetes.io/sriov-capable: 'true'
  numVfs: 4

```

Reply:
cc @wizhaoredhat @bn222 @vrindle

Reply:
Need to state that we recommend adding  `- ens7f0np0` before the upgrade.

Reply:
/remove-lgtm

Reply:
@bn222  adding `- ens7f0np0` before upgrade or after upgrade, both two are worked in my testing.